.travis.yml

```diff
@@ -1,29 +1,23 @@
 language: c
 env:
+###############################################################################
   # FIXME: If we want to check assembly too SYNTAX_ONLY needs to be 0
   # FIXME: We'd like to build with WARNINGS_AS_ERRORS=1 but there are too
   # many warnings firing right now
   # Note: Can't use CC variable as TravisCI will override it
-  - C_COMPILER=gcc-4.8 SYNTAX_ONLY=1 WARNINGS_AS_ERRORS=0 VERBOSE=0 SUPPRESS_WARNINGS=1 REPORT_CC_FILE=2
+  - C_COMPILER=gcc SYNTAX_ONLY=1 WARNINGS_AS_ERRORS=0 VERBOSE=0 SUPPRESS_WARNINGS=1 REPORT_CC_FILE=2
```

Comment:
That would be even better. However, it does not work for me:

```
c$ make
make[1]: Entering directory `array-examples'
..//Makefile.rules:121: *** Compiler "cc" was not identified as gcc or clang.  Stop.
make[1]: Leaving directory `array-examples'
make: *** [all] Error 1
```

I tried this first, but when it did not work I thought I have to look in ´.travis.yml` to see how I have to run the checks. So if there is some other way, please add it to the readme.